bootcamp/tutorials/quickstart/hybrid_demo.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<p>No need to install pymilvus[model] again. It was installed in the previous step.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1389/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>